packages/manager/apps/container/src/cookie-policy/CookiePolicy.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+import ovhCloudLogo from './assets/logo-ovhcloud.png';
+import links from './links';
+import './style.scss';
+
+const CookiePolicy = React.forwardRef((props, ref) => {
+  const { t } = useTranslation('cookie-policy');
+  const { ovhSubsidiary } = props.shell
+    .getPlugin('environment')
+    .getEnvironment().user;
+
+  const moreInfoLink = links.moreInfo[ovhSubsidiary];
+  const clickHereLink = links.changeOpinionLink[ovhSubsidiary];
+
+  const moreInfoHtml = () => {
```

Comment:
You can use `<Trans>` component in order to set html content into translation:
* doc: https://react.i18next.com/latest/trans-component
* example: https://github.com/ovh/manager/blob/feat%2Fproduct-nav-reshuffle-user-menu/packages/manager/apps/container/src/container/nav-reshuffle/header/user-account-menu/content.jsx#L45